package.json

```diff
@@ -16,7 +16,7 @@
     "test": "TZ=UTC vitest run",
     "test-watch": "TZ=UTC vitest",
     "test-coverage": "TZ=UTC vitest run --coverage",
-    "test-commit": "TZ=UTC vitest run --changed HEAD~1 --coverage",
+    "test-commit": "TZ=UTC LC_ALL=en_US.UTF-8 vitest run --changed HEAD~1 --coverage",
```

Comment:
Nice easy fix!  I suggest you add it to all the `test` yarn commands or the other commands will end up having the same problem.

Reply:
Sure thing!